src/components/ListDetails/ListDetails.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <div class="list-details">
+    <AddTaskButton />
+    <ListsSummary />
+    <ListTasks />
+  </div>
+</template>
+
+<script>
+import AddTaskButton from "./AddTaskButton.vue";
+import ListsSummary from "./ListSummary.vue";
```

Comment:
You renamed the component, but the import name remains the same.

Reply:
Updated